src/models/UserEmail.ts

```diff
@@ -75,6 +78,33 @@ export class UserEmail extends MySqlModel {
     return null;
   }
 
+  // Custom Insert logic
+  static async insert(
```

Comment:
Would prefer you add a `create` function here instead of overriding the base MySQLModel's `insert` function. 

See [this example on the Section model](https://github.com/CDLUC3/dmsp_backend_prototype/blob/59415e7fec31ffbe838b06117c4d28c47230bbdd/src/models/Section.ts#L50)

Reply:
There already is a `create` method on line 109, just below the function I added. The issue was that that method calls insert, without a valid token, and so could not get the userId from it to complete the insert.

Reply:
@briri alternatively, we could update the base model insert method to use the userId from the `obj` passed to it if token is null?

Reply:
most tables would be unlikely to have a `userId` field. Ok, I guess leave it as `insert` for now. Can you just make sure there's ample inline documentation explaining that we need to do this because in the context of registration, we don't have a valid context.token?